devito/arch/archinfo.py

```diff
@@ -1102,6 +1121,8 @@ class Blackwell(Hopper):
 
 class AmdDevice(Device):
 
+    thread_group_size = 64
```

Comment:
So intel device don't have this right?

Reply:
they should, but it's more difficult to reconcile things with SYCL 

anyway, SYCL doesn't use a CUDA-like pipeline, so we good